src/ui/wxWidgets/CreateShortcutDlg.cpp

```diff
@@ -188,17 +188,17 @@ void CreateShortcutDlg::CreateControls()
 
   auto StaticTextShortcutGroup = new wxStaticText(this, wxID_ANY, _("Group:"), wxDefaultPosition, wxDefaultSize, 0, _T("wxID_ANY"));
   FlexGridSizer1->Add(StaticTextShortcutGroup, 1, wxALL|wxALIGN_RIGHT|wxALIGN_CENTER_VERTICAL, 5);
-  m_ComboBoxShortcutGroup = new wxComboBox(this, ID_COMBOBOX1, wxEmptyString, wxDefaultPosition, wxDefaultSize, 0, nullptr, wxCB_DROPDOWN, wxDefaultValidator, _T("ID_COMBOBOX1"));
+  m_ComboBoxShortcutGroup = new wxComboBox(this, static_cast<wxWindowID>(ID_COMBOBOX1), wxEmptyString, wxDefaultPosition, wxDefaultSize, 0, nullptr, wxCB_DROPDOWN, wxDefaultValidator, _T("ID_COMBOBOX1"));
```

Comment:
Is it really necessary to have that many casts spread across the code? In some cases, wouldn't it be an option to adjust the data type instead?

Reply:
Yes, the various cast are introduced to come away from compiler warnings about implicit conversions.

Reply:
I think in this particular case the control ID variables can be declared with type `wxWindowID`, unless that would require casting them to `int` elsewhere?

Reply:
Declaration changed and static cast removed